README.md

```diff
@@ -1,2 +1,24 @@
-# Billing_and_Recurring_Payment_System
-It's a subscription based plan system where users can subscribe to different plans according to their needs and use the special features
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version
+
+* System dependencies
+
+* Configuration
+
+* Database creation
+
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
+
+* Deployment instructions
+
+* ...
```

Comment:
update the Readme file

Reply:
hn g asim bahi wo mn abhi thra bugs fix kr rha us k bd krni update